### DISCUSSION TITLE
Manga Manager Rework Beta

### DISCUSSION THREAD

**Discussion by ThePromidius** [MangaManagerORG/Manga-Manager#128] [Announcements]
Dear valued users,

I'm excited to finally share with you the beta version of the reworked software. When I started this project, it was just a personal learning exercise and wasn't intended to be shared. However, with your support and encouragement, I was able to turn it into something much more.

Along the way, I've learned a lot and been able to improve the software with your helpful bug reports. I'm grateful for your patience and can't wait for you to experience the improved functionality and user experience.

Please keep in mind that this is still an early stage beta release, and while the software has been thoroughly tested, there may still be some issues or bugs present. Your feedback and bug reports will be greatly appreciated as we work to further refine and improve the software.

Additionally, if you find the software useful, I've set up a donation option as a way to show your appreciation and support future development. Your generosity is greatly appreciated.

To summarize the changes, here's a list of key features:

* Select multiple files and preview the covers of each file
* Select a folder and open files recursively
* Bulk edit metadata for all files at once
* Changes are kept in one session, allowing for flexible editing
* Apply all changes (save to file) at once when done editing
* Edit cover or back cover from the metadata view itself
* Cover manager for batch editing of covers
* Online metadata scraping (Currently: AniList, MangaUpdates)
* Webp converter
* Error and warning log within the UI itself

Some upcoming/planned features. (Unordered list)

* Cover detector to remove similar covers/backcovers
* Bulk bump chapters
* Number/Volume parser from filename
* Merge chapter parts into a single chapter
* Native Web UI
* More sources
* Cover Downloader
* Manga Tagger Interface

Thank you for being a part of this journey with me.

<hr /><em>This discussion was created from the release <a href='https://github.com/MangaManagerORG/Manga-Manager/releases/tag/v1.0.0-beta.1'>Manga Manager Rework Beta</a>.</em>

**Comment by zjDrummond** [comment]
How do you use the webp converter?

**Reply by ThePromidius** [reply]
Hi there! Thanks for using MangaManager!
Latest release has a bug that does not display webp converter extension in the extensions tab. It's already fixed, however the release got halted due to having some issues with some optimizations i made to support multiprocessing. I planned to make a release this week so stay tuned.
